capabilities.json

```diff
@@ -63,6 +63,12 @@
             "displayName": "Extra information",
             "displayNameKey": "Role_ExtraInformation",
             "kind": "Grouping"
+        },
+        {
+            "name": "Tooltips",
+            "displayName": "Tooltips",
+            "displayNameKey": "Role_Tooltips",
+            "kind": "Measure"
```

Comment:
Have you tried to use `GropingAndMeasure`? Is that possible to allow users to use text data fields as well in tooltips?
IMHO text fields make sense.

Reply:
It is possible to use text data for tooltips. Not really sure why we need to use GropingAndMeasure in this case. We need a specific condition to use GropingAndMeasure